Source/WebCore/html/HTMLScriptElement.cpp

```diff
@@ -113,6 +114,45 @@ String HTMLScriptElement::crossOrigin() const
     return parseCORSSettingsAttribute(attributeWithoutSynchronization(crossoriginAttr));
 }
 
+ExceptionOr<void> HTMLScriptElement::setInnerTextForBinding(const std::variant<String, RefPtr<TrustedScript>>& value)
+{
+    ExceptionOr<String> stringValueHolder = WTF::switchOn(
+        value,
+        [this](const String& str) -> ExceptionOr<String> {
```

Comment:
WebKit coding style uses words, not abbreviations like `str`.

Reply:
This was marked as resolved, but was not resolved. Please don’t use `str` in the new code.

Reply:
Apologies for the oversight will make sure to fix this.